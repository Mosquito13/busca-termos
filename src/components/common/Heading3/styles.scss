.heading-3 {
    font-size: 2.5rem;
    font-weight: 600;
    color: currentColor;
}