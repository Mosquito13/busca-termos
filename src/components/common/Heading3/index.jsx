import PropTypes from 'prop-types';

import './styles.scss';

const Heading3 = ({ value }) => <h3 className="heading-3">{value}</h3>;

Heading3.propTypes = {
  /**
   * Valor do cabecalho
   */
  value: PropTypes.string
};

export default Heading3;
