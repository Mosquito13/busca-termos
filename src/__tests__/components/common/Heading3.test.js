import renderer from 'react-test-renderer';

import Heading3 from '../../../components/common/Heading3';

describe(`${Heading3.name}`, () => {
  describe('snapshots', () => {
    it('should render', () => {
      const tree = renderer
        .create(<Heading3 value="value for heading" />)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
