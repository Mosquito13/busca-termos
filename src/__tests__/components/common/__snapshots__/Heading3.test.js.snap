// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading3 snapshots should render 1`] = `
<h3
  className="heading-3"
>
  value for heading
</h3>
`;
